src/components/quotes.js

```diff
@@ -0,0 +1,57 @@
+import './Calculator.css';
+import React, { useEffect, useState } from 'react';
+
+const Quotes = () => {
+  const [quote, setQuote] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+
+  useEffect(() => {
+    const getQuotes = async () => {
+      setIsLoading(true);
+      setIsError(false);
+      try {
+        const res = await fetch(
+          'https://api.api-ninjas.com/v1/quotes?category=learning',
+          {
+            method: 'GET',
+            headers: {
+              'X-Api-Key': 'wifJwRXq2JXJebvDhx2hhA==2o6k7DNoMHsIgzzm',
+            },
+            contentType: 'application/json',
+          },
+        );
+        console.log(res);
```

Comment:
- [ ] According to the general JS requirements you should not commit console.log to your repo. It’s ugly, kills performance and can make confidential data visible to anyone using the browser tools to view your website.